spanner/google/cloud/spanner_v1/keyset.py

```diff
@@ -41,10 +41,10 @@ def __init__(self, start_open=None, start_closed=None,
         if not any([start_open, start_closed, end_open, end_closed]):
```

Comment:
Do you not want to check `is None` in this `any()` to conform to the way it's handled below?